### PR TITLE
merge master into abi3 branch to resolve conflicts

### PR DESCRIPTION
**Pull request by alex**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,13 +20,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
```
Comment (davidhewitt) [inline CHANGELOG.md:20]: Something's gone awry here - we've got two `## [Unreleased]` sections. Might be worth holding a sec to fix until I merge #1236 and maybe merge some of the other PRs so the changelog clashes get smoothed out.

**Review by davidhewitt — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (alex) [reply CHANGELOG.md:20]: I did that intentionally when I added the original changelog entry. This conflict didn't impact the changelog.

Comment (alex) [reply CHANGELOG.md:20]: Ahhh, I see what happened. I'll fix.

Comment (alex) [reply CHANGELOG.md:16]: Ok, better now.

**Review by davidhewitt — APPROVED** [review]
👍 thanks

**Comment by davidhewitt** [comment]
Ah interesting, looks like the fix to CI for MSRV has found something on this branch too! 😄

**Comment by alex** [comment]
Indeed it did. Will fix.

**Comment by davidhewitt** [comment]
Hmm that segfault is interesting 🤔

**Comment by alex** [comment]
If it only reproduces on that one build, I'm not sure what to make of it :-/

**Comment by davidhewitt** [comment]
I'm planning to drop Python 3.5 support very soon anyway (as part of 0.13). So if it's just that job let's merge this into abi3 branch anyway for now and I'll put up a PR to drop Python 3.5 support in the next day or two.

**Comment by alex** [comment]
Sounds great.

**Comment by alex** [comment]
yeah, just the one build :-/

**Comment by davidhewitt** [comment]
👍

**Pull request merged**